travis/run-tests-inside-guest

```diff
@@ -1,19 +1,23 @@
 #!/bin/bash
 set -x
 
+base="/root/go/src/github.com/dmacvicar"
```

Comment:
@inercia  would be nice if we can update also the script ``` travis/run-tests-inside-guest``` , this is used when we do 

```make test``` in dev mode :sunflower:

Reply:
Maybe if we shared several variables, but I think that is an overkill just for using one common variable...

Reply:
yop, i think we can leave with the 2 files for this time beeing

Reply:
Thanks!  I am hoping this resolves my timeout issues when using wait-for-lease.